### PR TITLE
RoadUserCountsHandler saves visualization

### PR DESCRIPTION
**Pull request by youralien**
Summary:

- Add /roadUserCounts route
- Created handler to save road user counts as an icon visualization
- Uses app_config to access the correct project directory
- TODO: Error Handling

### CODE REVIEW

File: app/handlers/roadUserCounts.py [hunk 1]
```diff
@@ -17,4 +21,20 @@ class RoadUserCountsHandler(tornado.web.RequestHandler):
     @apiError error_message The error message to display.
     """
     def post(self):
+        identifier = self.get_body_argument('identifier')
+        res = RoadUserCountsHandler.handler(identifier, )
```
Comment (runnersaw) [inline app/handlers/roadUserCounts.py]: What's with the `, )`?

Comment (runnersaw) [reply app/handlers/roadUserCounts.py]: Also, what does `res` become?

**Review by runnersaw — DISMISSED** [review]
I think just fix the hanging `, )`

**Comment by youralien** [comment]
Fixed the hanging `, )`.  Also, added an example of how status codes and error handling COULD work.  Of course, I will replace the `MyAppBaseHandler` with the handler that is created in a separate base handlers file.

**Review by jabb1123 — APPROVED** [review]
🚢 🇮🇹

**Pull request merged**